Sources/ArcGISToolkit/Common/AttachmentPreview.swift

```diff
@@ -70,6 +70,7 @@ struct AttachmentPreview: View {
         Carousel { computedCellSize, scrollToLeftAction in
             Group {
                 makeCarouselContent(for: computedCellSize)
+                    .transition(.asymmetric(insertion: .move(edge: .top), removal: .scale))
```

Comment:
This is my preference but please look at the different options and let me know what you think.

Reply:
| Move | Slide |
|:------:|:-----:|
|<video src="https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/3998072/8051fe01-050e-4d39-96a2-a04ec3d699ea" width="320" height="240" controls></video>|<video src="https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/3998072/2ad9e4a1-52cd-4c47-a015-656678f3e16f" width="320" height="240" controls></video>|

Reply:
I agree, slide does look better